package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "eslint-config-scality",
+    "name": "@scality/guidelines",
```

Comment:
For my understanding: why do we need a `package.json` ?

Reply:
I'm not sure I understood the question, do you mean why the `package.json` is used in the context of publishing npm packages? Or in general?

Reply:
Nevermind, it was a misunderstanding on my part. 
I meant specifically this `package.json` in the guidelines repo.
I somehow missread the path of the file and thought that it is specific to this action.